CHANGELOG.md

```diff
@@ -9,6 +9,15 @@ Future Todo List
 - Deprecate modal and orderCard component styles in next major version as unused.
```

Comment:
Can we actually remove these two files as mentioned here?

Reply:
Sorry – missed this one. I'll make sure I add this in when we update it in the future though (unless you get there first 😄 )